src/reference-manual/pathfinder.Rmd

```diff
@@ -0,0 +1,28 @@
+# Pathfinder
+
+The CmdStan method `pathfinder` runs the Pathfinder algorithm @zhang_pathfinder:2022.
```

Comment:
Drop line 3---we've avoided talking about interface specifics in the reference manual.  Just say that "Stan supports the Pathfinder algorithm of @zhang..."